src/templates/antora/antora.converter.ts

```diff
@@ -79,19 +76,27 @@ const cleanup = (str: string): string => {
   return filters.reduce((acc, filter) => filter(acc), str);
 };
 
-const getNameFromFullName = (name: string): string =>
-  name.split('.').slice(-1).join('');
-
 const getFilePathFromFullName = (name: string): string => {
   const filename = name.toLowerCase() === 'tinymce' ? 'tinymce.root' : name.toLowerCase();
-  return basePath + filename + '.adoc';
+  return 'apis/' + filename + '.adoc';
 };
 
-const generateTypeXref = (type: string): string =>
-  type.includes('tinymce', 0) ? 'xref:' + getFilePathFromFullName(type) + '[' + getNameFromFullName(type) + ']' : type;
+const getFilePathFromFullNameLegacy = (name: string): string => {
+  const folder = name.split('.').slice(0, -1).join('.') + '/';
```

Comment:
`name.split('.').slice(0, -1)` seems to be duplicated a couple times and looks to be related to the legacy logic... can we please pull that out to a separate function to make it clearer what it's doing?

Reply:
new function name `getNamespaceFromFullName`